packages/react-native-sdk/src/components/Livestream/LivestreamControls/LivestreamAudioControlButton.tsx

```diff
@@ -46,9 +46,9 @@ export const LivestreamAudioControlButton = () => {
         ]}
       >
         {!optimisticIsMute ? (
-          <Mic color={colors.static_white} />
```

Comment:
Changing this on purpose with different color as in the other use cases with Mic on/off.